components/brave_extension/extension/brave_extension/containers/braveShields.tsx

```diff
@@ -87,6 +88,16 @@ export default class Shields extends React.PureComponent<Props, State> {
     this.props.actions.shieldsReady()
   }
 
+  componentDidUpdate (prevProps: Props) {
+    // If current URL does not match current URL data in Shields,
```

Comment:
Didn't notice this earlier, this comment is out of date now, please update it, thank you.